### PR TITLE
[netatmo] Updated and added german translations

### PR DESCRIPTION
**Pull request by cweitkamp**
- Updated and added german translations
- Clean-up of xml-files

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties [hunk 1]
```diff
@@ -117,6 +129,9 @@ channel-type.netatmo.lastStatusStore.description = Zeigt den Zeitpunkt an, an de
 channel-type.netatmo.timeUtc.label = Zeitpunkt Messung
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties]: Just a reminder for myself: I maybe want to change this label to "Letzte Messung".

File: addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties [hunk 2]
```diff
@@ -117,6 +129,9 @@ channel-type.netatmo.lastStatusStore.description = Zeigt den Zeitpunkt an, an de
...
 channel-type.netatmo.timeUtc.description = Zeigt den Zeitpunkt an, an dem die letzte Datenmessung stattgefunden hat.
 
+channel-type.netatmo.lastMessage.label = Zeitpunkt Meldung
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties]: Just a reminder for myself: I maybe want to change this label to "Letzte Meldung".

**Comment by martinvw** [comment]
@CWempe you are doing so well ;-)

File: addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties [hunk 3]
```diff
@@ -80,7 +86,10 @@ channel-type.netatmo.dateMaxTemp.label = Zeitpunkt Max. Temperatur
 channel-type.netatmo.dateMaxTemp.description = Zeigt den Zeitpunkt an, an dem die maximale Temperatur gemessen wurde.
 
 channel-type.netatmo.temperatureTrend.label = Temperatur Trend
-channel-type.netatmo.temperatureTrend.description = Zeigt den Temperatur Trend (z.B. "up", "down" oder "stable") an.
+channel-type.netatmo.temperatureTrend.description = Zeigt den Temperatur Trend (z.B. "Steigend", "Steigend" oder "Fallend") an.
```
Comment (CWempe) [inline addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties]: This should be "Temperaturtrend" in one word.
"Steigend" is there twice.
"Stabil" is missing. ;)

File: addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties [hunk 4]
```diff
@@ -105,7 +114,10 @@ channel-type.netatmo.pressure.description = Zeigt den aktuellen Luftdruck (in mb
 channel-type.netatmo.pressure.state.pattern = %.1f mbar
 
 channel-type.netatmo.pressureTrend.label = Luftdruck Trend
-channel-type.netatmo.pressureTrend.description = Zeigt den Luftdruck Trend an (z.B. "up", "down" oder "stable").
+channel-type.netatmo.pressureTrend.description = Zeigt den Luftdruck Trend an (z.B. "Steigend", "Stabil" oder "Fallend").
```
Comment (CWempe) [inline addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties]: This should be "Luftdrucktrend" in one word.

File: addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties [hunk 5]
```diff
@@ -138,3 +153,32 @@ channel-type.netatmo.pressureUnit.description = Zeigt die gew
...
+channel-type.netatmo.WindStrength.description = Zeigt die aktuelle Windgeschwindigkeit (in km/h) an.
+
+channel-type.netatmo.GustAngle.label = B�en Richtung
+channel-type.netatmo.GustAngle.description = Zeigt die aktuelle B�en Richtung (in �) an.
+
+channel-type.netatmo.GustStrength.label = B�en Geschwindigkeit
+channel-type.netatmo.GustStrength.description = Zeigt die aktuelle B�en Geschwindigkeit (in km/h) an.
+
+channel-type.netatmo.healthindex.label = Health Index
+channel-type.netatmo.healthindex.description = Zeigt den Health Index (healthy, fine, fair, poor, unhealthy) an.
```
Comment (CWempe) [inline addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties]: I think "(healthy, fine, fair, poor, unhealthy)" should be translated, too. Right?

**Comment by cweitkamp** [comment]
@CWempe Thank you very much. I changed it.

**Comment by martinvw** [comment]
@cweitkamp @CWempe thanks!

**Pull request merged**